endpoints/openrtb2/auction.go

```diff
@@ -444,6 +444,10 @@ func fillAndValidateNative(n *openrtb.Native, impIndex int) error {
 }
 
 func validateNativeContextTypes(cType native.ContextType, cSubtype native.ContextSubType, impIndex int) error {
+	if cType == 0 {
+		// Context is only recommended, so none is a valid type.
+		return nil
```

Comment:
Should we return in here without also checking `cSubtype` first?

Reply:
If there is no context type, would subtype have any meaning?

Reply:
I guess not. LOL

Reply:
The `contextsubtype` is optional but does have defined ranges same as cType. Not sure if we want to add it in the future, but I would think that's a separate story.